library/psa_crypto_se.c

```diff
@@ -75,6 +90,123 @@ const psa_drv_se_t *psa_get_se_driver( psa_key_lifetime_t lifetime )
 
 
 
+/****************************************************************/
+/* Slot management */
+/****************************************************************/
+
+/** The type of bit vector elements. A bit vector is represented as an
+ * array of bit vector elements. */
+typedef unsigned bv_element_t;
+/* Number of bits per bit vector elements. Assumes no padding bits,
+ * which Mbed TLS checks in selftest.c. */
+#define BV_BITS_PER_ELEMENT ( sizeof( bv_element_t ) * CHAR_BIT )
+
+struct psa_drv_se_slot_usage_s
+{
+    size_t size; /** Number of elements of \c vector */
+    bv_element_t vector[1]; /* Should be vector[] but we don't allow C99 */
+};
+
+static psa_key_slot_number_t su_size(
```

Comment:
Secure Ulement? Slot Usage? Slow Umbrella? Use more descriptive name please, to improve readability.

Reply:
Moot because this chunk of code is no longer present.